### PR TITLE
Add ability to use alternative protractor config file after first execution

### PR DESCRIPTION
**Pull request by theandrewlane**
Not a breaking change! If ```protractorRetryConfig``` is not provided, current behavior does not change😁

**Usage Scenario** - Parallel (sharded) e2e execution: _Some specs fail, possibly due to concurrency issues or race conditions. User specifies alternative config (file or cli args) via ```protractorRetryConfig``` for protractor-flake to use after first execution. This config, for example, specifies different ```chromeOptions```, disables parallel execution, and runs a custom plugin via a lifecycle hook which notifies user of potential concurrency issue._

### CODE REVIEW

**Comment by NickTomlin** [comment]
Thanks for the PR! I am currently busy with family and work stuff but I will try to get to get a response Thursday or early next week.

File: test/unit/protractor.flake.config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// An example configuration file
+exports.config = {
```
Comment (NickTomlin) [inline test/unit/protractor.flake.config.js]: What about moving this to `test/unit/support` or `test/unit/support/fixtures`?

File: src/index.js [hunk 2]
```diff
@@ -56,6 +59,12 @@ export default function (options = {}, callback = function noop () {}) {
       protractorArgs = filterArgs(protractorArgs)
       protractorArgs.push('--specs', specFiles.join(','))
     }
+
+    // If an alternative protractor config is specified, pass it in at the end of protractorArgs
```
Comment (NickTomlin) [inline src/index.js]: Nit: I think the logic here is clear enough that we don't need a comment but that's also very subjective :)

**Review by NickTomlin — APPROVED** [review]
💃 

I left some minor comments but this looks great!

File: test/unit/protractor.flake.config.js (hunk 1 above)
Comment (theandrewlane) [reply test/unit/protractor.flake.config.js]: Yes sir!

File: src/index.js (hunk 2 above)
Comment (theandrewlane) [reply src/index.js]: You're right, probably unnecessary - I might remove this in my next commit 😁

**Comment by theandrewlane** [comment]
@NickTomlin: Changes pushed, ready for merge & publish 🤓

**Comment by NickTomlin** [comment]
This was released in `3.2.0`. Thanks!

**Pull request merged**